.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
Was this file supposed to be included?

Reply:
Yep, it was, but forgot to call it out in the PR description. Happy to keep it in?